lua/lsp-file-operations/will-delete.lua

```diff
@@ -0,0 +1,47 @@
+local utils = require("lsp-file-operations.utils")
+local log = require("lsp-file-operations.log")
+
+local M = {}
+
+local function getWorkspaceEdit(client, fname)
+  log.debug("going to get workspace edit")
```

Comment:
```suggestion
  log.debug("Going to get workspace edit workspace/willDeleteFiles")
```

Reply:
Updated debug statement in latest commit!